bash_completion.d/000_bash_completion_compat.bash

```diff
@@ -178,6 +178,14 @@ _realcommand()
     return $rc
 }
 
+# @deprecated set the `BASH_COMPLETION_*` variables directly instead.
+[[ -v COMP_FILEDIR_FALLBACK ]] &&
+    : "${BASH_COMPLETION_FILEDIR_FALLBACK:=$COMP_FILEDIR_FALLBACK}"
```

Comment:
```suggestion
    : "${BASH_COMPLETION_FILEDIR_FALLBACK=$COMP_FILEDIR_FALLBACK}"
```

When there are both settings `BASH_COMPLETION_FILEDIR_FALLBACK=` and `COMP_FILEDIR_FALLBACK=set`, I think `BASH_COMPLETION_FILEDIR_FALLBACK` should have higher precedence.

Reply:
Right! Addressed in 8e855a2c3057fb745d28e23728bb2d7edd182379.

Reply:
Thanks!